eclass/kernel-install.eclass

```diff
@@ -541,16 +544,18 @@ kernel-install_pkg_pretend() {
 
 	if ! use initramfs && ! has_version "${CATEGORY}/${PN}[-initramfs]"; then
 		ewarn
-		ewarn "WARNING: The standard configuration of the Gentoo distribution"
-		ewarn "kernels requires an initramfs! You have disabled the initramfs"
-		ewarn "USE flag and as a result dracut was not pulled in as a dependency."
-		ewarn "Please ensure that you are either overriding the standard"
-		ewarn "configuration or that an alternative initramfs generation plugin"
-		ewarn "is installed for your installkernel implementation!"
+		ewarn "WARNING: The standard Gentoo distribution kernel configuration is designed"
+		ewarn "to be used with an initramfs! Although possible, there is no guarantee"
+		ewarn "distribution kernels will boot without an initramfs."
 		ewarn
-		ewarn "This is an advanced use case, you are on your own to ensure"
-		ewarn "that your system is bootable!"
+		ewarn "The initramfs USE flag is disabled, and sys-kernel/installkernel is not"
+		ewarn "configured to use an initrd_generator, so an initramfs will not be built."
```

Comment:
```suggestion
		ewarn "force configured to use an initrd_generator, so an initramfs will not be built."
```

Maybe we can add here "force" or something similar, because installkernel may still be configured correctly, but disabling the flag disables that portage enforces that it is configured correctly.

Reply:
> Maybe we can add here "force" or something similar, because installkernel may still be configured correctly, but disabling the flag disables that portage enforces that it is configured correctly.

I think that's a good idea, but is force the right word? I want to be careful to not add words that could be confused with other portage features.  I wrote that referring to USE flags on the package, there's probably a better way to say something along the lines of "installkernel isn't set to USE an initramfs generator, so that configuration won't be set by portage, so it must be manually configured; more info here: <link>"

Reply:
Maybe "the package manager did not configure an initramfs generator"

It should be clear that toggleing the flag changes what portage does, not necessarily what installkernel does. The current wording implies that installkernel is not configured to use an initramfs generator, which is not necessarily true, we don't know about install.conf, we only know about the use flag dependency being absent.

Reply:
I propose the following:

```
		ewarn
		ewarn "WARNING: The default distribution kernel configuration is designed"
		ewarn "to be used with an initramfs! Although possible, there is no guarantee"
		ewarn "that distribution kernels will boot without an initramfs."
		ewarn
		ewarn "You have disabled the initramfs USE flag, and as a result the package manager
        ewarn "will not enforce the configuration of an initramfs generator in"
        ewarn "sys-kernel/installkernel."
        ewarn
		ewarn "If you wish to use a custom initramfs generator, then please ensure that" 
        ewarn "/sbin/installkernel is capable of calling it via a kernel installation hook,"
        ewarn "and is also configured to use it via /etc/kernel/install.conf."
        ewarn
        ewarn "If you wish to boot without an initramfs, then please ensure that"
        ewarn "all kernel drivers required to boot your system are built into the"
        ewarn "kernel by modifying the default distribution kernel configuration"
        ewarn "using /etc/kernel/config.d"
        ewarn
		ewarn "Please refer to the installkernel and distribution kernel documentation:"
		ewarn "    https://wiki.gentoo.org/wiki/Installkernel"
        ewarn "    https://wiki.gentoo.org/wiki/Project:Distribution_Kernel"
```

This I think makes it more clear that the action the user has taken (disable the initramfs flag) changes what portage does and not directly what installkernel does. And adds a bit more details on what one should do to ensure that the system remains bootable.